services/IBF-pipeline/pipeline/settings.py

```diff
@@ -207,9 +207,9 @@
 EMAIL_WITHOUT_TRIGGER = False
 
 # Notification email (True if hard-coded alternative for mailchimp is used)
-EMAIL_HARDCODE = False
+EMAIL_HARDCODE = True
```

Comment:
Please don't replace the default values for the variables `EMAIL_HARDCODE` and `EMAIL_LIST_HARDCODE`.